From 08edfad822c42062016313982d38b89aae261f28 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Fri, 5 Sep 2014 01:31:33 -0400
Subject: [iommu] vt-d: Mark functions as static in intel_irq_remapping.c

Message-id: <20140905013133.30867.30479.stgit@gir.stowe>
Patchwork-id: 91268
O-Subject: [RHEL7.1 PATCH 04/28] iommu/vt-d: Mark functions as static in intel_irq_remapping.c
Bugzilla: 1129808
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Dutile <ddutile@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1129808
Upstream Status: 6a7885c49b5b79c6e2e73db6fd9db9051b4a28f9

commit 6a7885c49b5b79c6e2e73db6fd9db9051b4a28f9
Author: Rashika Kheria <rashika.kheria@gmail.com>
Date:   Wed Dec 18 12:04:27 2013 +0530

    iommu/vt-d: Mark functions as static in intel_irq_remapping.c

    Mark functions int get_irte() and ir_dev_scope_init() as static in
    intel_irq_remapping.c because they are not used outside this file.

    This eliminates the following warnings in intel_irq_remapping.c:
    drivers/iommu/intel_irq_remapping.c:49:5: warning: no previous prototype for
    drivers/iommu/intel_irq_remapping.c:810:12: warning: no previous prototype f

    Signed-off-by: Rashika Kheria <rashika.kheria@gmail.com>
    Reviewed-by: Josh Triplett <josh@joshtriplett.org>
    Signed-off-by: Joerg Roedel <joro@8bytes.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/iommu/intel_irq_remapping.c b/drivers/iommu/intel_irq_remapping.c
index bab10b1..c988b8d 100644
--- a/drivers/iommu/intel_irq_remapping.c
+++ b/drivers/iommu/intel_irq_remapping.c
@@ -46,7 +46,7 @@ static struct irq_2_iommu *irq_2_iommu(unsigned int irq)
  return cfg ? &cfg->irq_2_iommu : NULL;
 }
 
-int get_irte(int irq, struct irte *entry)
+static int get_irte(int irq, struct irte *entry)
 {
  struct irq_2_iommu *irq_iommu = irq_2_iommu(irq);
  unsigned long flags;
@@ -807,7 +807,7 @@ int __init parse_ioapics_under_ir(void)
  return 1;
 }
 
-int __init ir_dev_scope_init(void)
+static int __init ir_dev_scope_init(void)
 {
  if (!irq_remapping_enabled)
   return 0;
-- 
1.7.1